Description: The way the statx function is defined is broken, and does not fall
back in case the syscall is not present, which is the case for mips64, which
runs Linux 4.9.
Upstream: rejected
URL: https://github.com/boostorg/filesystem/issues/173
diff --git a/libs/filesystem/build/Jamfile.v2 b/libs/filesystem/build/Jamfile.v2
index 132641a32..76547eb92 100644
--- a/libs/filesystem/build/Jamfile.v2
+++ b/libs/filesystem/build/Jamfile.v2
@@ -54,7 +54,6 @@ project boost/filesystem
       [ check-target-builds ../config//has_stat_st_birthtimensec "has stat::st_birthtimensec" : <define>BOOST_FILESYSTEM_HAS_STAT_ST_BIRTHTIMENSEC ]
       [ check-target-builds ../config//has_stat_st_birthtimespec "has stat::st_birthtimespec" : <define>BOOST_FILESYSTEM_HAS_STAT_ST_BIRTHTIMESPEC ]
       [ check-target-builds ../config//has_statx "has statx" : <define>BOOST_FILESYSTEM_HAS_STATX ]
-      [ check-target-builds ../config//has_statx_syscall "has statx syscall" : <define>BOOST_FILESYSTEM_HAS_STATX_SYSCALL ]
       <conditional>@select-windows-crypto-api
     : source-location ../src
     : usage-requirements # pass these requirement to dependents (i.e. users)
